gpMgmt/bin/generate-greenplum-path.sh

```diff
@@ -1,4 +1,31 @@
 #!/usr/bin/env bash
+# Notice from the Apache Cloudberry PPMC team as follows:
+# These files use the term 'greenplum' to maintain compatibility with
+# original versions of Apache Cloudberry, which was originally called
+# Greenplum. This usage does not relate to the VMware Tanzu Greenplum
+# product, neither do we imply that Apache Cloudberry (Incubating) is
+# affiliated with, endorsed by, or sponsored by Broadcom Inc.
+# See discussion at https://lists.apache.org/thread/b8o974mnnqk6zpy86dgll2pgqcvqgnwm
+
+cat <<"EOF"
+if [ -n "${PS1-}" ]; then
+    echo "Notice from the Apache Cloudberry PPMC team
```

Comment:
echo -e "\033[31mred text\033[0m", add color ?

Reply:
Good idea! How about using yellow instead of red?

Red usually indicates critical errors or failures. Yellow is typically used for warnings or important, non-critical info. Since this is an informational disclaimer, yellow can grab attention without implying an error.

Also, we can highlight the first line, not all lines, for a better visual experience.

Reply:
OK.

Reply:
IMHO: For this change, I would rather keep it a simple as possible and do not introduce the color functionality.

Reply:
Ok. That makes sense. Let's keep it simple.